source/features/show-github-action-next-time.tsx

```diff
@@ -0,0 +1,83 @@
+import './show-github-action-next-time.css';
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {parseCron} from '@cheap-glitch/mi-cron';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL, getRepoGQL, getCurrentBranch} from '../github-helpers';
+
+const getActionsSchedules = cache.function(async (): Promise<{[index: string]: string} | undefined> => {
+	const {repository: {object: {entries: actions}}} = await api.v4(
+		`repository(${getRepoGQL()}) {
+			object(expression: "${getCurrentBranch()}:.github/workflows") {
+				... on Tree {
+					entries {
+						object { ... on Blob { text } }
+					}
+				}
+			}
+		}
+	`);
+
+	if (!actions) {
+		return undefined;
+	}
+
+	const schedules: {[index: string]: string} = {};
+	for (const actionYaml of actions.map((action: {[index: string]: {[index: string]: string}}) => action.object.text)) {
+		const name = /^name:\s+['"]?(.+)['"]?/m.exec(actionYaml);
+		const cron = /^\s*-\scron:\s+['"](.+)['"]/m.exec(actionYaml);
```

Comment:
I don't think it's safe to use the `m` flag or else when will `.+` stop on an unquoted `name`?

```suggestion
		const name = /^name[:\s'"]+(.+)['"]?/.exec(actionYaml);
		const cron = /^\s*-\scron[:\s'"]+(.+)['"]/.exec(actionYaml);
```

This also covers `name : "name"`